opmon/cli.py

```diff
@@ -20,11 +20,13 @@
 from opmon.logging import LogConfiguration
 from opmon.metadata import Metadata
 from opmon.monitoring import Monitoring
+from opmon.utils import bq_normalize_name
 
 logger = logging.getLogger(__name__)
 
 
 DEFAULT_PLATFORM = "firefox_desktop"
+LOOKER_PREVIEW_URL = "https://mozilla.cloud.looker.com/dashboards/operational_monitoring::preview"
```

Comment:
This doesn't exist yet, but will get created once https://github.com/mozilla/looker-spoke-default/pull/457 gets merged